content/TrueCommand/SystemManagement/NASUsersAndGroups.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "NAS Users and Groups"
+weight: 40
+---
+
+TrueCommand lets you create NAS users and groups across multiple systems.
+
+## NAS Users
+
+### Adding a New User
+
+![NASUsersAndGroups1](/images/TrueCommand/2.0/NASUsersAndGroups1.png "Users and Groups")
+
+To add a NAS user to one or more systems, go to your dashboard and click the <i class="material-icons" aria-hidden="true" >more_vert</i> in a system's window, then select **Users and Groups**.
+
+![NASUsersAndGroups2](/images/TrueCommand/2.0/NASUsersAndGroups2.png "Users and Groups Screen")
+
+Click the **+ User** button to open the user creation wizard.
+
+{{< tabs "New NAS User" >}}
+{{< tab "Systems" >}}
+![NASUser1](/images/TrueCommand/2.0/NASUser1.png "Creating a New NAS User")
+
+Choose one or more systems for this user. Users and groups created across multiple systems will share Ids.
+
+Click **+ Add System** and select the systems you want to add the new user to, then click **NEXT**.
+{{< /tab >}}
+
+{{< tab "User/Groups" >}}
+![NASUser2](/images/TrueCommand/2.0/NASUser2.png "Creating a New NAS User")
+
+Configure the user's details. Optionally choose associated groups.
+
+Enter a *Username*, *Password*, *Email* (optional), and *Uid* (user ID). You can also select groups to associate the user with or create a new group using the **+ GROUP** button. 
+
+Once you are finished, click **NEXT**.
+{{< /tab >}}
+
+{{< tab "Home" >}}
+![NASUser3](/images/TrueCommand/2.0/NASUser3.png "Creating a New NAS User")
+
+Set the user's home directory. This step is optional.
+
+If you want the user to have a home directory, enter the path to the directory and set the default UNIX permissions, then click **NEXT**.
+{{< /tab >}}
+
+{{< tab "Authentication" >}}
+![NASUser4](/images/TrueCommand/2.0/NASUser4.png "Creating a New NAS User")
+
+Configure the user's authorization settings.
+
+You can enter or paste the user's public SSH key in the *Sshpubkey* field.
```

Comment:
change *Sshpubkey* to **Sshpubkey** if a UI element (above says it is the field name).